2023-08-05 16:49 ERROR Internal Server Error: /app/ 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
KeyError: 'chamberlain'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 23, in index
    return  render(request,'app/index.html',{'major':major})
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'chamberlain' is not a registered namespace
2023-08-05 16:50 ERROR Internal Server Error: /app/ 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
KeyError: 'chamberlain'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 23, in index
    return  render(request,'app/index.html',{'major':major})
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'chamberlain' is not a registered namespace
2023-08-05 16:50 ERROR Internal Server Error: /app/ 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 23, in index
    return  render(request,'app/index.html',{'major':major})
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/template/defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/urls/resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for '#' not found. '#' is not a valid view function or pattern name.
2023-08-05 16:52 WARNING Not Found: /favicon.ico 
2023-08-06 10:11 WARNING Not Found: / 
2023-08-06 10:11 WARNING Not Found: /favicon.ico 
2023-08-06 10:28 WARNING Not Found: /favicon.ico 
2023-08-06 10:28 WARNING Not Found: /favicon.ico 
2023-08-06 11:34 WARNING Not Found: /favicon.ico 
2023-08-06 11:34 WARNING Not Found: /apple-touch-icon-precomposed.png 
2023-08-06 11:34 WARNING Not Found: /apple-touch-icon.png 
2023-08-06 11:36 WARNING Not Found: /favicon.ico 
2023-08-06 11:36 WARNING Not Found: /favicon.ico 
2023-08-06 11:38 WARNING Not Found: / 
2023-08-06 12:21 WARNING Not Found: /favicon.ico 
2023-08-06 12:27 WARNING Not Found: /favicon.ico 
2023-08-06 12:31 WARNING Not Found: /favicon.ico 
2023-08-06 12:32 WARNING Not Found: /favicon.ico 
2023-08-06 14:02 WARNING Not Found: /favicon.ico 
2023-08-06 17:29 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 17:33 INFO /Users/yhlin/yhWorkspace/googvilla01/app/urls.py changed, reloading. 
2023-08-06 17:35 INFO /Users/yhlin/yhWorkspace/googvilla01/app/urls.py changed, reloading. 
2023-08-06 17:49 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 17:51 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 17:56 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 18:04 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 18:05 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 18:06 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 18:16 INFO <class 'list'> 
2023-08-06 18:17 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 18:18 INFO <class 'list'> 
2023-08-06 18:47 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 18:53 INFO <class 'list'> 
2023-08-06 18:53 INFO <class 'list'> 
2023-08-06 18:53 WARNING Not Found: /app/detail_view/2 
2023-08-06 18:53 INFO <class 'list'> 
2023-08-06 18:53 WARNING Not Found: /app/detail_view/2 
2023-08-06 18:53 INFO <class 'list'> 
2023-08-06 18:53 WARNING Not Found: /app/detail_view/4 
2023-08-06 18:54 INFO <class 'list'> 
2023-08-06 18:55 INFO <class 'list'> 
2023-08-06 18:55 INFO <class 'list'> 
2023-08-06 18:55 INFO <class 'list'> 
2023-08-06 18:55 INFO <class 'list'> 
2023-08-06 18:55 WARNING Not Found: /app/detail_view/2 
2023-08-06 19:00 INFO <class 'list'> 
2023-08-06 19:02 INFO <class 'list'> 
2023-08-06 19:02 WARNING Not Found: /app/detail_view/2 
2023-08-06 19:03 INFO <class 'list'> 
2023-08-06 19:03 INFO <class 'list'> 
2023-08-06 19:03 WARNING Not Found: /app/detail_view/2 
2023-08-06 19:04 INFO /Users/yhlin/yhWorkspace/googvilla01/app/urls.py changed, reloading. 
2023-08-06 19:14 INFO /Users/yhlin/yhWorkspace/googvilla01/app/models.py changed, reloading. 
2023-08-06 19:14 INFO /Users/yhlin/yhWorkspace/googvilla01/app/models.py changed, reloading. 
2023-08-06 19:28 INFO <class 'list'> 
2023-08-06 19:32 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 19:35 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 19:35 ERROR Internal Server Error: /app/detail_view/1 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 33, in detail_view
    item = get_list_or_404(DetailItem,major_id=id)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 109, in get_list_or_404
    obj_list = list(queryset.filter(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1426, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1236, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1724, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'major_id' into field. Choices are: comment, detail, id, sub_item, sub_item_id
2023-08-06 19:35 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 19:35 ERROR Internal Server Error: /app/detail_view/1 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 33, in detail_view
    item = get_list_or_404(DetailItem,major_id=id)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 109, in get_list_or_404
    obj_list = list(queryset.filter(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1426, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1236, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1724, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'major_id' into field. Choices are: comment, detail, id, sub_item, sub_item_id
2023-08-06 19:36 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 19:36 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 19:37 ERROR Internal Server Error: /app/detail_view/1 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/app/views.py", line 33, in detail_view
    item = get_list_or_404(DetailItem,subitem_id=id)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/shortcuts.py", line 109, in get_list_or_404
    obj_list = list(queryset.filter(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1426, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1236, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/Users/yhlin/yhWorkspace/googvilla01/venv/lib/python3.10/site-packages/django/db/models/sql/query.py", line 1724, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'subitem_id' into field. Choices are: comment, detail, id, sub_item, sub_item_id
2023-08-06 19:38 INFO /Users/yhlin/yhWorkspace/googvilla01/app/views.py changed, reloading. 
2023-08-06 19:38 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 19:38 INFO [<DetailItem: 碎餐>, <DetailItem: 打菜>, <DetailItem: 菜單管理>, <DetailItem: 餐食統計>, <DetailItem: 其他>] 
2023-08-06 22:22 WARNING Not Found: /favicon.ico 
2023-08-06 22:22 INFO [<SubItem: 備餐>, <SubItem: 陪餐>, <SubItem: 餐廳支援>] 
2023-08-06 22:22 INFO [<DetailItem: 碎餐>, <DetailItem: 打菜>, <DetailItem: 菜單管理>, <DetailItem: 餐食統計>, <DetailItem: 其他>] 
2023-08-06 22:24 INFO [<DetailItem: 碎餐>, <DetailItem: 打菜>, <DetailItem: 菜單管理>, <DetailItem: 餐食統計>, <DetailItem: 其他>] 
2023-08-06 22:25 INFO [<DetailItem: 碎餐>, <DetailItem: 打菜>, <DetailItem: 菜單管理>, <DetailItem: 餐食統計>, <DetailItem: 其他>] 
2023-08-06 22:25 INFO [<DetailItem: 碎餐>, <DetailItem: 打菜>, <DetailItem: 菜單管理>, <DetailItem: 餐食統計>, <DetailItem: 其他>] 
